plugins/operator-accumulator/src/main/scala/com/stratio/sparkta/plugin/operator/accumulator/AccumulatorOperator.scala

```diff
@@ -20,14 +20,19 @@ import java.io.{Serializable => JSerializable}
 
 import com.stratio.sparkta.sdk.TypeOp._
 import com.stratio.sparkta.sdk.{TypeOp, _}
+import org.apache.spark.sql.types.StructType
 
 import scala.util.Try
 
-class AccumulatorOperator(name: String, properties: Map[String, JSerializable]) extends Operator(name, properties)
+class AccumulatorOperator(name: String,
+                          schema: StructType,
+                          properties: Map[String, JSerializable]) extends Operator(name, schema, properties)
 with OperatorProcessMapAsAny with Associative {
 
   final val Separator = " "
 
+  val inputSchema = schema
```

Comment:
This val can be removed in every Operator. OperatorProcessMapAsAny should be the one that extends Operator